entrypoint.d/50-config-generate

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "${CONFIG_BUILD}" = true ]; then
```

Comment:
This variable is a build arg, not an env. It means it will not be available at run time, so you better `test -f $OPENERP_SERVER`